src/Umbraco.Web.UI.NetCore/Umbraco.Web.UI.NetCore.csproj

```diff
@@ -21,6 +21,7 @@
 
   <ItemGroup>
     <Folder Include="App_Plugins" />
+    <Folder Include="Views" />
```

Comment:
Don't think we want this one do we? won't cause issues for now though. Everything looks good, i'll merge